superset/utils/webdriver.py

```diff
@@ -198,10 +202,11 @@ def get_screenshot(
             ]
             logger.debug("Wait %i seconds for chart animation", selenium_animation_wait)
             sleep(selenium_animation_wait)
-            logger.debug(
-                "Taking a PNG screenshot of url %s as user %s",
+            logger.info(
+                "Taking a PNG screenshot of url %s as user %s with execution_id: %s",
```

Comment:
@craig-rueda I resolved a conflict here from a recent change of yours. I was hoping to get this execution id into our production logs. Do you think it's something that we should only log locally?

Reply:
I think this should be `debug`, as it's cluttering things unnecessarily. We can leverage the logging context if you wanted to add `execution_id` to the rendered log's JSON, which will land in the logging aggregator as a field

Reply:
Oh, and that's something that you can filter on? That sounds like a better implementation overall than adding it to the message field.